### PR TITLE
Add a servlet that returns the login status of the user on main page.

### PR DESCRIPTION
**Pull request by ak47na**
script.js will use the servlet to change the content on the main page depending on the login status of users.

### CODE REVIEW

File: portfolio/pom.xml [hunk 1]
```diff
@@ -36,6 +36,12 @@
      <version>1.9.59</version>
     </dependency>
 
+    <dependency>
```
Comment (alexcepoi) [inline portfolio/pom.xml]: This seems a duplicate from above. Is this intentional?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -96,12 +96,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
...
     for (Entity comment: results.asIterable()) {
-      comments.add((String)comment.getProperty("message"));
-
-      -- limit;
       if (limit == 0) {
         break;
       }
+      -- limit;
+      
+      comments.add((String)comment.getProperty("message"));
```
Comment (alexcepoi) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:104]: These changes seem unrelated? Refrain from bundling together unrelated changes (I understand the temptation).
There's nothing wrong to having a very small PR for a quick bugfix.

**Review by alexcepoi — APPROVED** [review]

File: portfolio/pom.xml (hunk 1 above)
Comment (ak47na) [reply portfolio/pom.xml]: No, thank you for the notice.
